packages/cli/src/test/integration/integration_test.ts

```diff
@@ -249,7 +270,7 @@ suite('integration tests', function() {
       }));
     });
 
-    test('test the lit-element "shop" template', async function() {
+    test.only('test the lit-element "shop" template', async function() {
```

Comment:
Make sure to remove this before merging ;)

Reply:
:X :X :X